pipeline/filters/standard-output.md

```diff
@@ -1,37 +1,35 @@
-# Standard Output
+The stdout filter plugin allows printing to the standard output the data flowed through the filter plugin, which can be very useful while debugging.
 
-The **stdout** output plugin allows to print to the standard output the data received through the _input_ plugin. Their usage is very simple as follows:
+The plugin has no configuration parameters, is very simple to use.
 
-## Configuration Parameters
+## Command Line
 
-| Key | Description | default |
-| :--- | :--- | :--- |
-| Format | Specify the data format to be printed. Supported formats are _msgpack_ _json_, _json\_lines_ and _json\_stream_. | msgpack |
-| json\_date\_key | Specify the name of the date field in output | date |
-| json\_date\_format | Specify the format of the date. Supported formats are _double_,  _iso8601_ \(eg: _2018-05-30T09:39:52.000681Z_\) and _epoch_. | double |
+```
+$ bin/fluent-bit -i cpu -F stdout -m '*' -o null
+```
 
-### Command Line
 
-```bash
-$ bin/fluent-bit -i cpu -o stdout -v
-```
 
-We have specified to gather [CPU](https://github.com/fluent/fluent-bit-docs/tree/ddc1cf3d996966b9db39f8784596c8b7132b4d5b/pipeline/input/cpu.md) usage metrics and print them out to the standard output in a human readable way:
+We have specified to gather [CPU](https://github.com/fluent/fluent-bit-docs/tree/ddc1cf3d996966b9db39f8784596c8b7132b4d5b/pipeline/input/cpu.md) usage metrics and print them out in a human-readable way when they flow through the stdout plugin, setting `-m '*'`to print all the metrics.
```

Comment:
The link CPU is broken.
The path is `../inputs/cpu-metrics.md`

Reply:
I think the description about `-m` is not needed.
Because `-m` is not unique option for filter_stdout plugin.